src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php

```diff
@@ -12,13 +12,15 @@
 namespace Symfony\Bridge\PhpUnit;
 
 use Doctrine\Common\Annotations\AnnotationRegistry;
+use PHPUnit\Framework\BaseTestListener;
+use PHPUnit\Framework\TestCase;
 
 /**
  * Collects and replays skipped tests.
  *
  * @author Nicolas Grekas <p@tchwork.com>
  */
-class SymfonyTestsListener extends \PHPUnit_Framework_BaseTestListener
+class SymfonyTestsListener extends BaseTestListener
```

Comment:
What will using namespaces here provide at all on 2.8?
The bridge before 3.3 is still incompatible with phpuni 6 - using namespaces will only forbid it's use with non-namespaced versions of phpunit. 
#21668 is unrelated - it's only for master.